jans-cedarling/cedarling/src/common/cedar_schema/test_files/test_schema_with_record_ctx.json

```diff
@@ -0,0 +1,150 @@
+                        "iss": {
+                            "type": "EntityOrCommon",
+                            "name": "TrustedIssuer"
+                        }
+                    }
+                }
+            },
+            "Issue": {
+                "shape": {
+                    "type": "Record",
+                    "attributes": {
+                        "country": {
+                            "type": "EntityOrCommon",
+                            "name": "String"
+                        },
+                        "org_id": {
+                            "type": "EntityOrCommon",
+                            "name": "String"
+                        }
+                    }
+                }
+            },
+            "Role": {},
+            "TrustedIssuer": {
+                "shape": {
+                    "type": "Record",
+                    "attributes": {
+                        "issuer_entity_id": {
+                            "type": "EntityOrCommon",
+                            "name": "Url"
+                        }
+                    }
+                }
+            },
+            "User": {
+                "memberOfTypes": [
+                    "Role"
+                ],
+                "shape": {
+                    "type": "Record",
+                    "attributes": {
+                        "country": {
+                            "type": "EntityOrCommon",
+                            "name": "String"
+                        },
+                        "email": {
+                            "type": "EntityOrCommon",
+                            "name": "String"
+                        },
+                        "sub": {
+                            "type": "EntityOrCommon",
+                            "name": "String"
+                        },
+                        "username": {
+                            "type": "EntityOrCommon",
+                            "name": "String"
+                        }
+                    }
+                }
+            },
+            "Workload": {
+                "shape": {
+                    "type": "Record",
+                    "attributes": {
+                        "client_id": {
+                            "type": "EntityOrCommon",
+                            "name": "String"
+                        },
+                        "iss": {
+                            "type": "EntityOrCommon",
+                            "name": "TrustedIssuer"
+                        },
+                        "name": {
+                            "type": "EntityOrCommon",
+                            "name": "String"
+                        },
+                        "org_id": {
+                            "type": "EntityOrCommon",
+                            "name": "String"
+                        }
+                    }
+                }
+            }
+        },
+        "actions": {
+            "Update": {
```

Comment:
You can add to this file action with other name for example `Update2` with context as type. 
Please avoid using two separate files for this purpose if can.

Reply:
used one test file here [589d564](https://github.com/JanssenProject/jans/pull/10387/commits/589d56449f643f1402a086192758ccd47115eed1)